ocaml/idl/datamodel.ml

```diff
@@ -1712,7 +1712,8 @@ module Sr_stat = struct
       ~doccomments:[]
       ~messages_default_allowed_roles:(Some []) (* No messages, so no roles allowed to use them *)
       ~contents:
-        [ (* The uuid is not needed here and only adds inconvenience. *)
+        [
+          field ~qualifier:DynamicRO ~lifecycle ~ty:(Option String) "uuid" "Uuid that uniquely identifies this SR, if one is available.";
```

Comment:
This is a new field introduced by this branch, so no need for additional version specifiers. We should probably squash it when merging to feature branch though.